test/unit/demutils.js

```diff
@@ -12,39 +12,66 @@ import OBB from 'Renderer/OBB';
 import LayerUpdateState from 'Layer/LayerUpdateState';
 import DEMUtils from 'Utils/DEMUtils';
 import { RasterElevationTile } from 'Renderer/RasterTile';
+import sinon from 'sinon';
 import Renderer from './bootstrap';
 
+const BIL_ROWS = 256;
+const BIL_COLS = 256;
+function createBilData(elevation = 1) {
+    return new Float32Array(new Array(BIL_COLS * BIL_ROWS).fill(elevation));
+}
+
 describe('DemUtils', function () {
     const renderer = new Renderer();
     const placement = { coord: new Coordinates('EPSG:4326', 1.5, 43), zoom: 10 };
     const viewer = new GlobeView(renderer.domElement, placement, { renderer });
 
-    const source = new WMTSSource({
-        format: 'image/x-bil;bits=32',
-        crs: 'EPSG:4326',
-        url: 'https://data.geopf.fr/wmts?',
-        name: 'ELEVATION.ELEVATIONGRIDCOVERAGE.SRTM3',
-        tileMatrixSet: 'WGS84G',
-        networkOptions: process.env.HTTPS_PROXY ? { agent: new HttpsProxyAgent(process.env.HTTPS_PROXY) } : {
-            // referrerPolicy: 'origin-when-cross-origin',
-            crossOrigin: 'anonymous',
-            // referrer: 'http://localhost:8080/examples/view_3d_map.html',
-        },
-    });
-    source.url = 'https://github.com/iTowns/iTowns2-sample-data/blob/master/dem3_3_8.bil?raw=true';
-    const elevationlayer = new ElevationLayer('worldelevation', { source });
+    let elevationlayer;
+    let context;
+    let stubSuppFetcher;
+    const ELEVATION = 300;
+
+    before(function () {
+        stubSuppFetcher = sinon.stub(supportedFetchers, 'get')
+            .callsFake((format) => {
+                if (format === 'image/x-bil;bits=32') {
```

Comment:
```suggestion
        stubSuppFetcher = sinon.stub(supportedFetchers, 'get');
        stubSuppFetcher.withArgs('image/x-bil;bits=32')
            .callsFake(() => {
                const floatArray = createBilData(ELEVATION);
```
Or is it meant to throw an error ? This way the mock happen only for this format. WDYT ?

Reply:
I didn't know that this method was existing. I did the change.
If the tests are well done, there shouldn't be a nned for throwing an error.